datafusion/proto/tests/cases/roundtrip_physical_plan.rs

```diff
@@ -589,34 +586,6 @@ async fn roundtrip_parquet_exec_with_table_partition_cols() -> Result<()> {
     roundtrip_test(Arc::new(ParquetExec::new(scan_config, None, None)))
 }
 
-#[test]
-fn roundtrip_builtin_scalar_function() -> Result<()> {
```

Comment:
I removed this test because we are going to elimiate buit-in functions

Reply:
Until we have completed the migration, can you please simply change to use a different built in function so we don't lose coverage?

Reply:
done